docs/technical-details/release-notes/v4-tucker/4.7.0.md

```diff
@@ -0,0 +1,13 @@
+---
+title: 4.7.0
+sidebar_position: 59392
```

Comment:
```suggestion
sidebar_position: 59299
```

Start at `59299` so that we can continue to make 4.6 patch release within the bounds of `399 - 300`.